src/studio/README.md

```diff
@@ -91,6 +99,7 @@ npm run gulp-install-deps
 ```
 
 On MacOS you need two extra steps:
+
   1. change the RepositoryLocation in src/studio/src/designer/backend/appsettings.json to
 
       ```json
```

Comment:
Could you also fix this block by removing the extra 'tick' on the end (block end). It has four ticks instead of three.